README.md

```diff
@@ -15,6 +15,13 @@ The specification contains many examples to start learning Flux.
 
 Flux is currently available in InfluxDB 1.7 and 2.0 or through the REPL that can be compiled from this repository.
 
+To build flux, first install the `pkg-config` utility, and
+ensure the GNU `pkg-config` utility is also installed.
+
+```
+$ go get github.com/influxdata/pkg-config
```

Comment:
This won't work correctly since it will change the `go.mod` file. It will work correctly though if you are outside of the project directory.